platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -4196,6 +4196,9 @@
 
             var values = $scope.currentRecord.fields;
             angular.forEach (values, function(value, key) {
+                if(value.type.defaultName === "str" && value.value === ""){
```

Comment:
Why are we extracting the default name parameter from the type for this check? Isn't there any more appropriate field?